src/TypeSpec.Extension/Emitter.Csharp/package.json

```diff
@@ -50,17 +50,17 @@
     },
     "dependencies": {
         "@autorest/csharp": "3.0.0-beta.20230309.1",
-        "@azure-tools/typespec-autorest": "0.30.0",
-        "@azure-tools/typespec-azure-core": "0.30.0",
-        "@azure-tools/typespec-client-generator-core": "0.30.0",
-        "@typespec/compiler": "0.44.0",
+        "@azure-tools/typespec-autorest": "0.31.0",
+        "@azure-tools/typespec-azure-core": "0.31.0",
+        "@azure-tools/typespec-client-generator-core": "0.31.0",
+        "@typespec/compiler": "0.45.0",
         "@typespec/eslint-config-typespec": "0.7.0",
-        "@typespec/eslint-plugin": "0.44.0",
-        "@typespec/http": "0.44.0",
-        "@typespec/library-linter": "0.44.0",
-        "@typespec/openapi": "0.44.0",
-        "@typespec/rest": "0.44.0",
-        "@typespec/versioning": "0.44.0",
+        "@typespec/eslint-plugin": "0.45.0",
+        "@typespec/http": "0.45.0",
+        "@typespec/library-linter": "0.45.0",
+        "@typespec/openapi": "0.45.0",
```

Comment:
Please remove `@typespec/openapi`  and `@azure-tools/typespec-autorest` aslo. Thanks